### DISCUSSION TITLE
Data card slot- progress bar specs

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1223] [Proposals]
### Goal

Especificar como debe comportarse el card slot de progress cuando no cabe y debe pasar a 2 lineas
![image](https://user-images.githubusercontent.com/95479574/235899011-d92c850f-a4b1-4b5d-bcc3-8a1a44450807.png)

### Context

Mientras analizábamos el impacto de aumentar el espacio del _next visible element_ en los carrousels, detectamos que en pantallas pequeñas o con textos largos, la data cards con el slot de progress se rompía

![image](https://user-images.githubusercontent.com/95479574/235899318-49e4000a-09e7-4d7f-bc74-4b0076ea2e24.png)

### Impact

Ahora que vamos a aumentar el espacio destinado a visualizar el _next visible element_ vamos a encontrarnos más con este problema y necesitamos tener definido como debe comportarse este slot cuando pasa a 2 lineas. 

### Discussion Questions

-¿Esta definición debe proveerla Mistica o al ser un slot deben realizarse unas specs adhoc para esto?
-Al ser un componente que se utiliza mucho, ¿tiene sentido un Novum extended de las _cards slots_ que tenemos disponibles en Figma?
![image](https://user-images.githubusercontent.com/95479574/235899929-b5977095-2f83-4d0a-9742-d53e04ea2250.png)

**Comment by lauraocana** [comment]
Hola @MontseTelefonica, los componentes que introduzcamos en el slot son responsabilidad de los equipos que los introducen, esta composición no es un componente de Mistica como tal, así que la responsabilidad de la definición, mantenimiento y evolución es nuestra. Podemos ver si tiene sentido redefinir este componente como Novum Extended, pero es más tema nuestro que de Mistica.

**Reply by MontseTelefonica** [reply]
Vale, pues como este slot de progress se usa mucho creo que es carne de definirse bien como extended, y de paso aprovechar para insertar las mejoras que tenemos detectadas.

**Reply by yceballost** [reply]
Exacto, como dice Laura, lo que hay en la librería de Mística son ejemplos de construcciones de slot.. estábamos hablando en el equipo de eliminar los ejemplos para evitar justo este tipo de malentendidos.

Como bien comentáis, eso debería existir como un componente en la librería extended de cada producto (en este caso Novum Extended). También de esta manera tendríais el control absoluto de cambiar cosas sin necesidad de pasar por Mística.

Os parece bien si lo sacamos de la librería? le ponemos capita roja de deprecated y lo sustituís por el que creéis en la extended. (lo deprecamos para final de quarter para no reventaros ahora nada)

**Reply by MontseTelefonica** [reply]
Por mi perfecto, creamos tarea para esto y nos estrenamos definiendo un componente Novum Extended @lauraocana ?

**Reply by lauraocana** [reply]
A por ello ;)

**Reply by yceballost** [reply]
Cierro discussion y creo ticket! gracias majas
